HelloWeb/src/main/resources/META-INF/native-image/filter-file.json

```diff
@@ -0,0 +1,8 @@
+{ "rules": [
```

Comment:
I don't think we need any of the native-image config files from runagent, do we?

Reply:
This application can also run on Desktop and these files are required?

Reply:
I have removed instructions for Desktop and related files.